### ISSUE TITLE
Add created_by and updated_by fields to base model

### ISSUE THREAD

**Issue by sainak** [ohcnetwork/care#1697]
Add created_by and updated_by fields to base model to track user that made changes to the objects

**Comment by AnkurPrabhu** [comment]
Hey can i work on this one ?

**Comment by sainak** [comment]
sure

**Comment by AnkurPrabhu** [comment]
@sainak just wanted to confirm this will have 
1) a migration file for the addition of two fields in class BaseModel 
2) all the changes where any object of a class extended by BaseModel is either created or modified right ?

**Comment by sainak** [comment]
> @sainak just wanted to confirm this will have
> 
>     1. a migration file for the addition of two fields in class BaseModel
> 
>     2. all the changes where any object of a class extended by BaseModel is either created or modified right ?

Yes, and also override perform create/update function of viewsets

**Comment by AnkurPrabhu** [comment]
@sainak will the migrations files be a separate pr ?

**Comment by AnkurPrabhu** [comment]
@sainak can you pls take a look at the initial pr just wanted to confirm if im in the right direction or not.
made the migration field and updated the created_by and updated_by fields for some viewsets, once you approve the approach will do it for all the other viewsets

**Comment by AnkurPrabhu** [comment]
@sainak  i have updated the pr with all the necessary changes pls take a look

**Comment by Harsh-D-2004** [comment]
is this still open??
